letsdrill.rb

```diff
@@ -1,13 +1,29 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ grade = ""
+  grade_ranges = {
+  	"A" => 101,
+  	"B" => 90,
+  	"C" => 80,
+  	"D" => 70,
+  	"F" => 60,
+  }
 
+  grade_ranges.each do |letter, upper_bound|
+  	if (integer < upper_bound)
+  		grade = letter
+  	end
+  end
+  return grade
```

Comment:
it's standard to omit the `return` at the end of a method, since ruby automatically returns the last value